governance/polls/Ratification Poll for MIP50 Direct Deposit Module Amendment (MIP4c2-SP23) - August 8, 2022.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Ratification Poll for MIP50 Direct Deposit Module Amendment (MIP4c2-SP23) - August 8, 2022
+summary: MIP50 was too specific to lending protocols. I’ve updated it for the V2 rewrite which covers all external protocols that allow users to deposit / withdraw DAI.
+discussion_link: https://forum.makerdao.com/t/mip4c2-sp23-mip50-direct-deposit-module-amendment/16387
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2022-08-08T16:00:00
+end_date: 2022-08-22T16:00:00
+---
+# Ratification Poll for MIP50 Direct Deposit Module Amendment (MIP4c2-SP23) - August 8, 2022
+
+The Governance Facilitators have placed a ratification poll into the [voting system](https://vote.makerdao.com/polling) as part of the responsibilities defined in [MIP51](https://mips.makerdao.com/mips/details/MIP51). This Governance [Poll](https://community-development.makerdao.com/en/learn/governance/on-chain-gov) will be active for fourteen days beginning on Monday, August 8 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to MIP4c2-SP23 being accepted and implemented in the Maker Protocol.
+
+A brief summary of this proposal has been provided by the MIP Author and is shown below:
+
+*MIP50 was too specific to lending protocols. I’ve updated it for the V2 rewrite which covers all external protocols that allow users to deposit / withdraw DAI. I’ve updated the MIP to remove details of the targeting mechanism as that is no longer important.*
```

Comment:
I suggest placing quotation marks around these summaries, in addition to italicizing